bin/hdfcoinc/pycbc_page_foreground

```diff
@@ -99,11 +99,15 @@ if args.output_file.endswith('.html'):
     mass2 = fortrigs.get_bankfile_array('mass2')
     spin1z = fortrigs.get_bankfile_array('spin1z')
     spin2z = fortrigs.get_bankfile_array('spin2z')
-    time1 = fortrigs.get_coincfile_array('time1')
-    time2 = fortrigs.get_coincfile_array('time2')
+    time = fortrigs.get_end_time()
 
     if h_num_rm is not None and h_iterations is not None \
             and h_num_rm >= 0 and h_iterations != 0:
+        # FIXME: Currently heirarchical removal is not in the multi-detector
+        #        code. If this functionality is added, and we want to make this
```

Comment:
code --> workflow
HR is done in the add_statmap code, but is just not yet implemented on the results pages

Reply:
There's no hierarchical removal in the input files here. If there was I could look to put it in here ...

Reply:
[referring to chunk 20 o2 run for multiifo testing] 
The combine_statmap one doesn't, but the H1L1-EXCLUDE_ZEROLAG one does

The HV background _does_ extend past the stat value of the loud foreground trigger

It should be doing HR in add_statmap based on the exclusive IFAR. For these tests, could you use the HL exclude zerolag and I'll try to find if theres a problem with add_statmap